build_tools/ci/run_matmul_test.sh

```diff
@@ -485,7 +492,7 @@ run_matmul_test \
     --peano_install_path "${PEANO}" \
     --mlir_aie_install_path "${MLIR_AIE_INSTALL}" \
     --vitis_path  "${VITIS}" \
-    --pipeline "pad-pack" \
+    --tile_pipeline "pad-pack" \
```

Comment:
I've been trying to make this specific test explicitly list all the options, please add

```
--lower_to_aie_pipeline "air" \
```

as one additional line.

Reply:
Done